lib/cretonne/src/verifier.rs

```diff
@@ -350,7 +351,7 @@ impl<'a> Verifier<'a> {
         Ok(())
     }
 
-    fn cfg_integrity(&self, ebb: Ebb) -> Result<()> {
+    fn control_flow_integrity(&self, ebb: Ebb) -> Result<()> {
```

Comment:
Maybe we should just delete this function?

Assuming that we trust the CFG implementation to be correct, it isn't really adding anything. We have specific CFG tests both as unit tests and `print-cfg` file tests. The value of the verifier is to check that the CFG stored in the context has been kept up to data with any program changes made by optimizations.